### ISSUE TITLE
Serbian localisation

### ISSUE THREAD

**Issue by VeskoRadic** [JulietaUla/Montserrat#112]
In the regular style, only the letter **б** is different from the Russian one. The five letters (**б, г, д, п, т**) differ in italics.
![image](https://user-images.githubusercontent.com/63882439/116315551-fc113180-a7b0-11eb-843d-66529900ab2e.png)

![image](https://user-images.githubusercontent.com/63882439/116314247-24982c00-a7af-11eb-8246-9a0f85208303.png)

As for italics, the letter **г** does not have the correct form.
Also, the letters **ш** and **т** should be similar, but in the current edition they are different.
![image](https://user-images.githubusercontent.com/63882439/116314880-fc5cfd00-a7af-11eb-9492-f418e9e89380.png)

I respect artistic freedom and originality, but the current form really catches the eye. I saw the font on several sites and immediately had the feeling that something was not right.
I apologize for the poor English.

**Comment by solmatas** [comment]
Hi @VeskoRadic,
thank you for the feedback.

The Montserrat project has a set of glyphs with localized forms for Serbian and Bulgarian.
I made an image of the groups of letters set in italics that you are pointing out.

There is a locl feature that you can activate if you prefer Serbian or Bulgarian forms.

<img width="330" alt="Localized Forms" src="https://user-images.githubusercontent.com/8105496/116334639-d9454400-a7d5-11eb-9f53-e534079996b0.png">

**Comment by VeskoRadic** [comment]
I apologize if I wasn't clear.

I know that there are localized glyphs but they are not correct. In the first picture I have attached, you can see what they should look like, regular style on left side, italics on right.

The second image shows what the current regular-style localization looks like and what it should look like. At the moment, it seems that regular and italics are mixed, which is not good.

As for italics, the word **ш** should be like **т** without the line above - in this case the styles also seem mixed. As for the letter **г**, it should be a bit more rounded in the lower part, as you can see in the first picture – it looks wrong like this.

**Comment by solmatas** [comment]
There are three different designs for some of the glyphs:
1 - the default set (let's call it Russian)
2 - the Bulgarian set
3- the Serbian set
I am attaching a new image with italics and upright. 
It would be great if you can use that image and delete what shouldn't be there and leave only your suggestions for the Serbian localized forms for italics and upright. 

Thank you. 
<img width="725" alt="Serbian" src="https://user-images.githubusercontent.com/8105496/116340427-7193f680-a7df-11eb-8e12-2d6072cbde4d.png">

**Comment by VeskoRadic** [comment]
Again, as a Serb, I think that I know how look Serbian letters, and I trying to tell you that design of localised glyphs is not correct and that the letters look as if the styles are mixed. Look again at this picture, on the left side is regular, on the right italic. 

![116315551-fc113180-a7b0-11eb-843d-66529900ab2e](https://user-images.githubusercontent.com/63882439/116350649-93e33f80-a7f2-11eb-8e4b-10e5a43e6cd4.png)
 
The current design is not correct in either style. If I'm not clear enough, since English is not my main language, maybe you should look at some fonts that have the correct localization of the localization (Cormorant, Noto, Vollkorn, Spectral…)

I'm sorry if you think this is anything but a desire to help.

**Comment by solmatas** [comment]
Hi there,
I do appreciate your feedback.
I follow the yellow image you attached, but the italic comments you suggested are not clear for me. That is the reason why I sent the last image with all the options for those glyphs asking for your input using that set of the Montserrat glyphs. 
This will be helpful to be sure that we are on the same page about the modifications.
Once I have this I can share this with the rest of the design team and the native type designer involved in the project.
Thank you very much.

**Comment by VeskoRadic** [comment]
As I  said, I'm not native English speaker and I can understand that I am not speaking clearly.

I hope that the references I mentioned and pictures above will be useful. I'm glad if I can help make the font even better.

**Comment by VeskoRadic** [comment]
I couldn't get to the computer earlier, I don't even know if I need to write again, but I will try to clarify a few things that I mentioned earlier.

I hope that now there are no doubts about the regular style, that is, that only the letter **б** should have a localized variant.

![image](https://user-images.githubusercontent.com/63882439/116457336-903bd100-a863-11eb-9193-bc3d738ae395.png)
![image](https://user-images.githubusercontent.com/63882439/116457584-de50d480-a863-11eb-8ec4-e62c45812a65.png)

But if the font is to be upright cursive, then some more characters would have to be changed so that there would be no impression of mixed styles. In that sense, I mentioned the comparison between **ш** and **т** and that would then apply to all styles.

As for the italics, I really wasn’t the clearest. I don’t want to question your ideas but it seems to me that the font doesn’t have a uniform look. I mentioned a few glyphs, but I don't think they're the only ones. At the moment it seems to me that there is an uneven appearance, that is, some glyphs are slanted, some are italicized. I hope that from the attached pictures you can see what I mean.

![image](https://user-images.githubusercontent.com/63882439/116458368-e0fff980-a864-11eb-96c5-451bf046b8d9.png)

What should it look like if it were italic:
![image](https://user-images.githubusercontent.com/63882439/116458595-1f95b400-a865-11eb-9c02-f49a6186b36b.png)

… or slanted font:
![image](https://user-images.githubusercontent.com/63882439/116458706-39cf9200-a865-11eb-9260-d50d1958a0d7.png)

Basically, Russian and Serbian Cyrillic differ only in a few letters. If the Russian version should be slanted, then the Serbian version should not be in italics. If Russian should be italicized after all, then their glyphs are not good either.
Below is a picture showing variants of letters that differ in Russian and Serbian Cyrillic (in the middle is the Bulgarian) in both styles:

![image](https://user-images.githubusercontent.com/63882439/116461018-05110a00-a868-11eb-85da-0aa2765f53a9.png)

**Comment by solmatas** [comment]
Dear Vesko,

Thank you very much for your feedback about the Serbian glyphs. This has been very helpful. 🙏 
We changed the source files and exported fonts following your recommendations.

<img width="1164" alt="changes" src="https://user-images.githubusercontent.com/8105496/122739512-598e9f80-d283-11eb-9aa1-1d1005ed5d16.png">


The new version is 7.220.
I am closing this issue and if you have any comments to improve the project, please reopen it.
Thanks again!
